orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -26,12 +27,9 @@ def __init__(self, model):
         self._export_mode = torch.onnx.TrainingMode.TRAINING
 
     @staticmethod
-    def execution_session_run_forward(execution_session, onnx_model, device, *inputs):
+    def execution_session_run_forward(execution_session, onnx_model, *inputs):
         """Runs the forward graph on execution_session with given model inputs and device"""
 
-        # Assert that the input and model device match
-        _utils._check_same_device(device, "Input argument to forward", *inputs)
```

Comment:
Why delete this instead of checking with `if self._skip_check.is_set(_SkipCheck.SKIP_CHECK_DEVICE) == False` as in the other instances below?

Reply:
I can't access self here, so moved it to forward() at line 122, just like we had in backward()